src/redis_cmd.cc

```diff
@@ -4388,7 +4241,11 @@ Status LookupCommand(const std::string &cmd_name,
   if (cmd_factory == commands.end()) {
     return Status(Status::RedisUnknownCmd);
   }
-  *cmd = cmd_factory->second.factory();
+  auto redisCmd = cmd_factory->second;
+  *cmd = redisCmd.factory();
+  (*cmd)->SetArity(redisCmd.arity);
+  (*cmd)->SetIsWrite(redisCmd.is_write);
+  (*cmd)->SetName(redisCmd.name);
```

Comment:
for `arity, is_write, or others`, i think they are attributes of `RedisCommand` instead of `Commander`, i think commander just like a executor. so could we set one member to refer `RedisCommand`.

Reply:
nice advice, but seems `RedisCommand` wasn't a good name if we want to make it as the `Commader` member. We need a better name like `CommandAttributes`, how do you think.

Reply:
+10086